src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -28,7 +28,8 @@
     'azure-cli-core',
     'azure-mgmt-web==0.31.0',
     # v1.17 breaks on wildcard cert https://github.com/shazow/urllib3/issues/981
-    'urllib3[secure]==1.16'
+    'urllib3[secure]==1.16',
+    'pyOpenSSL>=0.16.2',
```

Comment:
This looks like a typo as the current latest version is 16.2.0.
Also in `setup.py` it's recommended to not specify a version unless there's a specific feature you use from 16.2.0? 
azure-cli-iot` and `azure-cli-keyvault` both have the dependency specified as `'pyOpenSSL'` with no specific version.